File: KCD/HMMasterMapCellCommand.m
//
//  HMMasterMapCellCommand.m
//  KCD
//
//  Created by Hori,Masaki on 2014/02/17.
//  Copyright (c) 2014年 Hori,Masaki. All rights reserved.
//

#import "HMMasterMapCellCommand.h"

@implementation HMMasterMapCellCommand
- (NSString *)dataKey
{
	return @"api_data.api_mst_mapcell";
}
- (NSArray *)ignoreKeys
{
	return @[@"api_req_shiptype", @"api_link_no"];
}
- (void)execute
{
	[self commitJSONToEntityNamed:@"MasterMapCell"];
}
@end

File: KCD/HMMasterMapCellCommand.swift
//
//  HMMasterMapCellCommand.swift
//  KCD
//
//  Created by Hori,Masaki on 2015/01/11.
//  Copyright (c) 2015年 Hori,Masaki. All rights reserved.
//

import Cocoa

class HMMasterMapCellCommand: HMJSONCommand {
	override var dataKey: String {
		return "api_data.api_mst_mapcell"
	}
	override var ignoreKeys: [AnyObject] {
		return ["api_req_shiptype", "api_link_no"]
	}
	override func execute() {
		super.commitJSONToEntityNamed("MasterMapCell")
	}
}
